File: platforms/ios/www/plugins/custom-plugin-panorama/www/panorama.js
//cordova.define("custom-plugin-panorama.Panorama", function(require, exports, module) {
var exec = require('cordova/exec');

var Panorama = {

  start: function() {
    exec(null, null, 'Panorama', 'start', []);
  }
};
module.exports = Panorama;

//});

File: plugins/custom-plugin-panorama/src/ios/Panorama.swift
import Foundation

@objc(Panorama) class Panorama : CDVPlugin {

    func start(command: CDVInvokedUrlCommand) {        
        print("exec from plugin")
        let v : ViewController = ViewController()
        self.viewController.addChildViewController(v)
        self.viewController.view.addSubview(v.view)
    }
}
